sc/social/like/plugins/facebook/templates/metadata.pt

```diff
@@ -3,7 +3,7 @@
 </tal:comment>
 <tal:fb condition="not:view/is_plone_5">
   <meta property="og:site_name" tal:attributes="content view/portal_title" />
-  <meta property="og:url" tal:attributes="content view/url" />
+  <meta tal:condition="view/canonical_url" property="og:url" tal:attributes="content view/canonical_url" />
```

Comment:
If I understand right og:url needs to be here with the current url if the behavior is not enabled.

So the condition should be removed and the method view.canonical_url return the current url in this case.